src/main/java/com/simibubi/create/compat/Mods.java

```diff
@@ -77,7 +77,8 @@ public boolean contains(ItemLike entry) {
 	 * @return a boolean of whether the mod is loaded or not based on mod id
 	 */
 	public boolean isLoaded() {
-		return ModList.get().isLoaded(id);
+		ModList modList = ModList.get();
+		return modList != null && modList.isLoaded(id);
```

Comment:
`ModList.get()` from LexForge and NeoForge is not marked nullable; if you do have an error about NPE here, that should be reported to their side instead of implementing a workaround here.